.changeset/famous-pets-notice.md

```diff
@@ -0,0 +1,27 @@
+---
+"@jpmorganchase/uitk-core": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Rename Drop Target characteristic to Target
+
+```diff
+- --uitk-dropTarget-borderStyle
+- --uitk-dropTarget-borderStyle-hover
+- --uitk-dropTarget-borderStyle-disabled
+- --uitk-dropTarget-borderWidth
+- --uitk-dropTarget-borderWidth-hover
+- --uitk-dropTarget-borderWidth-disabled
+- --uitk-dropTarget-cursor-disabled
+- --uitk-dropTarget-background-hover
++ --uitk-target-background-hover
++ --uitk-target-borderStyle
++ --uitk-target-borderStyle-hover
++ --uitk-target-borderStyle-disabled
++ --uitk-target-borderWidth
++ --uitk-target-borderWidth-hover
++ --uitk-target-borderWidth-disabled
++ --uitk-target-cursor-disabled
++ --uitk-target-borderColor-hover: var(--uitk-palette-interact-border-hover)
++ --uitk-target-borderStyle-hover: solid
+```
```

Comment:
```suggestion
```diff
- --uitk-dropTarget-background-hover
- --uitk-dropTarget-borderStyle
- --uitk-dropTarget-borderStyle-hover: dashed;
- --uitk-dropTarget-borderStyle-disabled
- --uitk-dropTarget-borderWidth
- --uitk-dropTarget-borderWidth-hover
- --uitk-dropTarget-borderWidth-disabled
- --uitk-dropTarget-cursor-disabled
+ --uitk-target-background-hover
+ --uitk-target-borderStyle
+ --uitk-target-borderStyle-hover: solid
+ --uitk-target-borderStyle-disabled
+ --uitk-target-borderWidth
+ --uitk-target-borderWidth-hover
+ --uitk-target-borderWidth-disabled
+ --uitk-target-cursor-disabled
+ --uitk-target-borderColor-hover: var(--uitk-palette-interact-border-hover)
```
```

Just makes the order consistent adds the original valud for boderStyle hover